File: models/Message.js
module.exports = class Message {
  constructor({ id, user, channelId, content, createdAt }) {
    this.id = id;
    this.user = user;
    this.content = content;
    this.channelId = channelId;
    this.createdAt = createdAt;
  }
};

File: services/worker/src/Models/Message.cs
using System;
using System.Threading.Tasks;

namespace bot.Models
{
    public class Message
    {
        public string Id;
        public User User;
        public Chat Chat;
        public string Content;
        public DateTimeOffset CreatedAt;
        public async Task Reply(string content)
        {
            await Chat.SendMessage(content);
        }
    }
}
